tests/simple/simple.go

```diff
@@ -99,9 +99,42 @@ func Match(t *spb.SimpleTest, actual *exprpb.ExprValue) error {
 //	2) Map comparisons ignore order.
 func MatchValue(tag string, expected *exprpb.Value, actual *exprpb.Value) error {
 	// TODO write normalized comparator.
```

Comment:
Edit the TODO - what remains is just to make all NaNs compare equal.

Reply:
Not quite what I meant. The old version of the Godoc here was fine, defining the two normalizations that MatchValue should do. What needs updating is the TODO on line 100 - change it to "TODO make all NaNs compare equal".